packages/v4/src/content/accessibility/avatar/avatar.md

```diff
@@ -0,0 +1,22 @@
+---
+id: Avatar
+section: components
+---
+
+An **avatar** is a visual used to represent a user. It may contain an image or a placeholder graphic. Typical usage is to represent the current user in the masthead.
+
+**Keyboard users** shouldn’t be able to interact with the avatar on its own. Thus, the user should not be able to focus on the avatar using **Tab** to move forward and **Tab + Shift** to move backwards through interactive elements.
+
+**Screen reader users** should be able to navigate to the avatar and it should have alternative text, since it’s an image element: `aria-label` is the most common choice.
```

Comment:
Would this be `alt` instead? I think this line and the below recommendations are slightly confusing together/clash.